lib/OpenQA/CacheService/Model/Cache.pm

```diff
@@ -113,18 +107,16 @@ sub get_asset ($self, $host, $job, $type, $asset) {
 
     my $start;
     my $options = {
```

Comment:
Likely one of the nested subs defined here are responsible for the test failure you've mentioned.